### PR TITLE
Add syntax highlighting

### PR DESCRIPTION
**Pull request by jarofghosts**
Alright! So, I tried a few different methods to accomplish this. Using brucedown takes a whopping 15-20 seconds (or sometimes more) to render. It also seems to hog system resources during that time (on a 2013 MBP I experience lag). I have opted instead to use marked + highlight.js to emulate github syntax highlighting. The only downside that I have seen is _this_ method doesn't render code comments correctly (or at least not how github does it). If you would prefer, however, I will use the other method.

### CODE REVIEW

**Comment by DTrejo** [comment]
Perfect is the enemy of good, right? Nice work & feel free to push this to master & publish :)

File: readmetree.js [hunk 1]
```diff
@@ -47,16 +57,32 @@ function renderFiles(err, files) {
...
       + moduleName + '</a>'
+    content[shortpath] = style + nav 
+
+    loadReadme(path.resolve(root, readme), shortpath, moduleName)
   })
 
-  var url = 'http://localhost:'+PORT
-  console.log(url+' is serving all your readmes')
-  exec('open '+url)
+  function loadReadme(readme, shortpath, name) {
```
Comment (DTrejo) [inline readmetree.js:71]: if you want to take async as a dependency, that's totally okay (for the filecount stuff).

could also use named functions to reduce the nesting, but that's up to you, it's clean enough as-is

Comment (jarofghosts) [inline readmetree.js:71]: yeah, i opted for the named function route as i don't really care for `async`. i'll move `addReadme` to its enclosing scope, but the rest of the nesting is mostly necessary.

Comment (DTrejo) [inline readmetree.js:71]: I didn't read super closely; sounds good to me!

On Tue, Mar 11, 2014 at 5:10 PM, jesse keane notifications@github.comwrote:

> In readmetree.js:
> 
> >    })
> > -  var url = 'http://localhost:'+PORT
> > -  console.log(url+' is serving all your readmes')
> > -  exec('open '+url)
> > -  function loadReadme(readme, shortpath, name) {
> 
> yeah, i opted for the named function route as i don't really care for
> async. i'll move addReadme to its enclosing scope, but the rest of the
> nesting is mostly necessary.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/DTrejo/readmetree/pull/9/files#r10500786
> .

## 

See my projects at dtrejo.com

**Pull request closed without merging**